107/src/main/java/org/ehcache/jsr107/EhcacheCachingProvider.java

```diff
@@ -113,9 +143,14 @@ public Eh107CacheManager getCacheManager(URI uri, Configuration config) {
    * @return a cache manager
    */
   public Eh107CacheManager getCacheManager(URI uri, Configuration config, Properties properties) {
+    return getCacheManager(new ConfigSupplier(uri, config), properties);
+  }
+
+  public Eh107CacheManager getCacheManager(ConfigSupplier configSupplier, Properties properties) {
```

Comment:
I've made the visibility changes, not sure why github hasn't noticed but oh well